src/main/java/com/coinbase/prime/balances/ListEntityBalancesRequest.java

```diff
@@ -0,0 +1,109 @@
+package com.coinbase.prime.balances;
+
+import com.coinbase.prime.model.enums.BalanceType;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class ListEntityBalancesRequest {
```

Comment:
We don't need to do this for this pr, but pagination functionality should be defined in an abstract base class.

Reply:
I'd rather not as generation will be harder if we use abstract base classes